package-lock.json

```diff
@@ -9551,28 +9551,27 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": false,
+          "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
```

Comment:
I didn't see these changes going from `0.8 -> 1.0`—do you know how they got here going backward?

Reply:
...because that's another packages dependency?

Reply:
I did a fresh npm install so it probably bumped some stuff. I wouldn't comb through the package-lock manually 😆

Reply:
It's still good to gut-check it.

Yes, probably the new install, I was just curious as these changes alone "shouldn't" affect the lock file.